app/views/webpay/create.html.erb

```diff
@@ -6,12 +6,12 @@
   <label for="buy_order">
     Orden de compra
   </label>
-  <input id="buy_order" name="buy_order" value="123456"/>
+  <input id="buy_order" name="buy_order" value="<%='12345' + rand(1000).to_s %>"/>
```

Comment:
Maybe a timestamp will have less chance of duplicating buy orders?

Reply:
Good idea 👍